File: week7(13_3_24)/audio_samplerate.m
clc;
clear all;
close all;

% Read audio file
[audio, sampleRate] = audioread('/home/pvtrmrt/Documents/LABS/practice/week7(13_3_24)/bee-flying-loop-42287.mp3');
%audio:-t's typically a one-dimensional array where each element represents a sample of the audio signal.
% Display information
disp(['Sample rate: ' num2str(sampleRate) ' Hz']);
disp(['Number of samples: ' num2str(length(audio))]);

File: week7(13_3_24)/audio_samplerate.py
from scipy.io import wavfile

# Read audio file and get sample rate (x) and audio data (f)
x, f = wavfile.read('/home/pvtrmrt/Documents/LABS/practice/week8(27_3_24)/whisper-trail-2ogg-14429.wav')

# Display sample rate and number of samples
print('Sample rate:', x, 'Hz')
print('Number of samples:', len(f))
